packages/compass-e2e-tests/helpers/commands/wait-for-connection-result.ts

```diff
@@ -21,24 +21,38 @@ export async function waitForConnectionResult(
     selector = TEST_COMPASS_WEB
       ? '[data-testid="workspace-tab-button"][title=Databases]'
       : TEST_MULTIPLE_CONNECTIONS
-      ? Selectors.SidebarTreeItems
+      ? Selectors.Multiple.ConnectionItemConnected
       : Selectors.MyQueriesList;
   } else {
     // TODO(COMPASS-7600): this doesn't support compass-web yet, but also isn't
     // encountered yet
-    selector = Selectors.ConnectionFormErrorMessage;
+    // TODO(COMPASS-7397): as explained below, for multiple connections we
```

Comment:
I used this ticket number a lot because I figured it is blocked on at least that. Let me know if there is something better to use.